src/tir/schedule/ir_comparator.h

```diff
@@ -110,6 +110,48 @@ class TensorizeComparator : public ExprComparator, public StmtComparator {
   std::unordered_map<ObjectRef, ObjectRef, ObjectPtrHash, ObjectPtrEqual> equal_map_;
 };
 
+/*! \brief IR comparator for auto tensorization. Extract correspondence between the IR of the
+ *         workload and the tensor intrin.
+ */
+class AutoTensorizeExtractor : public TensorizeComparator {
```

Comment:
The name of this class is a bit confusing: It is derived from "Comparator", and it's main job does seem to be comparing things. But it is named "Extractor".

To me, the "extract" aspect of this class seems like a by-product. How about just call it "AutoTensorizeComparator"?

Also, it's better to document how this class is different from `TensorizeComparator`, in terms of IR comparison. Some of the overrides look identical to the base class ones.